### DISCUSSION TITLE
Congrats

### DISCUSSION THREAD

**Discussion by Tape-Worm** [dbalsom/marty#63] [General]
Hey Cow. Looks like you got more than just Alley Cat up and running. This is really awesome, I'm really glad to see it come together for you!

Tape_Worm

**Comment by dbalsom** [comment]
hey bud! come back to Discord, we miss you!

**Reply by Tape-Worm** [reply]
Sadly, I don't think I'll ever be back on discord. It really didn't do my mental health any favors, so I decided to get the hell out of dodge when things started getting really rough for me. Also, I don't even have the invite link anymore, so I have no means of reconnecting anyway.

That said, your work is awesome and I'm so glad you stuck with it. Hell, I'll probably have to consult your code to see if I'm doing shit right in Rust :)

Oh, and just FYI (if you don't already know). JetBrains is apparently releasing a Rust-specific IDE: [https://www.jetbrains.com/rust/](https://www.jetbrains.com/rust/), I'm waiting for it to come out of preview before I jump back into Rust myself.

**Comment by Tape-Worm** [comment]
So, due to needing to get some info about some stuff I use, I had to re-join Discord.  If you want to add me, my username is "REDACTED DUE TO PARANOIA"  I'm not on there often, but I do check it a couple of times a month.

I now await the trillions of spam bots sending me messages for publishing this on an open forum.

**Reply by dbalsom** [reply]
check your friend requests!!

**Reply by Tape-Worm** [reply]
Jesus, took you long enough :P

**Reply by Tape-Worm** [reply]
Actually, I don't see anything?  You sure you added the paranoia tentacle at the end of the name right?

**Reply by dbalsom** [reply]
no, no I did not.
